### PR TITLE
README: Add detailed contributions information

### PR DESCRIPTION
**Pull request by agherzan**
Signed-off-by: Andrei Gherzan <andrei@gherzan.com>

### CODE REVIEW

**Comment by twoerner** [comment]
nformation -> information

File: README
Comment (twoerner) [inline README]: too many "people"

**Review by twoerner — CHANGES_REQUESTED** [review]

File: README
Comment (agherzan) [reply README]: Let all the people review! Copy paste is not an amazing method of doing stuff.

Comment (agherzan) [reply README]: @twoerner fixed. Thanks.

**Comment by agherzan** [comment]
@twoerner looks good now?

**Comment by twoerner** [comment]
@agherzan yes, looks great, thanks 👍

**Comment by agherzan** [comment]
@twoerner let's unblock the PR with a approved review :)

**Review by twoerner — APPROVED** [review]
LGTM

**Comment by twoerner** [comment]
For me, the github page says: "At least one approved review is required by reviewers with write access." Since I don't have write access to this repository, I assumed that wasn't for me.

**Comment by agherzan** [comment]
@twoerner that makes sense.

**Pull request merged**
